README.md

```diff
@@ -83,46 +83,43 @@ All releases on the same Major version will be guaranteed to have backward compa
 
 ## Usage
 
-OSV-Scanner collects a list of dependencies and versions that are used in your project, before matching this list against the OSV database via the [OSV.dev API](https://osv.dev#use-the-api). To build the list of dependencies, you can point OSV-Scanner at your project directory, or manually pass in the path to individual manifest files.
+OSV-scanner parses lockfiles, SBOMs, and git directories to determine your project's open source dependencies. These dependencies are matched against the OSV database via the [OSV.dev API](https://osv.dev#use-the-api) and known vulnerabilities are returned to you in the output. 
```

Comment:
We've stylized this tool as "OSV-Scanner" in many places. Any particular reason to change this to "OSV-scanner" with the lowercasing?

Reply:
Nope. That is me learning the ropes.